### ISSUE TITLE
Mermaid charts should have accessible names and descriptions

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/orangelight#3477]
#### In firefox:

1. Go to https://github.com/pulibrary/orangelight/blob/main/docs/requests/form_diagram.md
2. Right-click on the flowchart.
3. Select "Accessibility properties"
4. Note that the diagram node (corresponding to the mermaid SVG) has no accessible name or description, and that there is an alert "Content with images must be labeled."

#### In Chrome:

1. Go to https://github.com/pulibrary/orangelight/blob/main/docs/requests/form_diagram.md
2. Select the `<svg>` element in the dev tools elements tab.
3. Choose the Accessibility tab
4. Note that under "Computer Properties", Name is set to an empty string.

### URL

There are three affected URLs:

- https://github.com/pulibrary/orangelight/blob/main/docs/requests/README.md
- https://github.com/pulibrary/orangelight/blob/main/docs/requests/form_diagram.md
- https://github.com/pulibrary/orangelight/blob/main/docs/alma_orangelight_login.md

### Issues

There is no accessible alternative to these diagrams, nor is there even an accessible name for a screen reader user to know what the diagrams are about.

### Severity

Not urgent, since it only affects people using the docs.

### WCAG Conformance level (A, AA, AAA)

Level A [WCAG 2.1 SC 1.1.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content): All non-text content that is presented to the user has a text alternative that serves the equivalent purpose

### User impact

The diagrams are meaningless to a screen reader user.

#### Accessible name

You can add an accessible name to an `<svg>` element with a [`<title>` element](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title).  There are two ways to do this in mermaid:
1. Add the title to the front matter.  This also makes the title available to sighted users.
````
```mermaid
---
title: My Accessible Title
---
flowchart LR
  id
```
````
1. Add an `accTitle` to the chart.  This is not shown to sighted users.
````
```mermaid
flowchart LR
  accTitle: My Accessible Title
  id
```
````

#### Accessible description

You can add an accessible name to an `<svg>` element with a [`<desc>` element](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/desc).  To do this in mermaid, use `accDescr`:

````
```mermaid
flowchart LR
  accTitle: My Accessible Title
  accDescr: An id node is by itself, without any other elements in the workflow.
  id
```
````

### Acceptance criteria
- [ ] I can successfully access the accessible names of these charts in Voice Over in Safari.

### Further reading

* [Mermaid accessibility guide](https://mermaid.js.org/config/accessibility.html#accessible-title)

**Comment by christinach** [comment]
completed
